geoips/interfaces/module_based/filename_formatters.py

```diff
@@ -53,20 +53,20 @@ class FilenameFormattersInterface(BaseModuleInterface):
         "standard_metadata": ["metadata_dir", "metadata_type", "basedir"],
     }
 
-    def find_duplicates(self, *args, **kwargs):
-        """Find duplicate files."""
-        try:
-            func = self.get_plugin_attr(name, "find_duplicates")
-        except AttributeError:
-            raise AttributeError(
-                f'Plugin {name} does not have a "find_duplicates" function.'
-            )
+    # def find_duplicates(self, *args, **kwargs):
```

Comment:
Please leave a comment as to why this was commented out

Reply:
added that comment, as well as to the other functions that were commented out